src/main/java/com/scottlogic/practice/mealscheduler/Services/MealService.java

```diff
@@ -2,6 +2,8 @@
 
 import com.scottlogic.practice.mealscheduler.Models.Meal;
 import com.scottlogic.practice.mealscheduler.Repositories.MealRepo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
I know very little about logging, just checking this isn't the library that has the recent famous security issue?

Reply:
Yes, that's the one. It has been removed in recent releases. For a commercial project, we would need to thoroughly check, not to leave an open vulnerability.